app/api/admin/queue-health/route.ts

```
@@ -0,0 +1,227 @@
+        });
+
+        if (middlewareError) {
+            return NextResponse.json(
+                { error: "Authentication failed" },
+                { status: 401 }
+            );
+        }
+
+        switch (action) {
+            case "cleanup":
+                const keepCompleted = body.keepCompleted || 100;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)